k2/python/k2/rnnt_loss.py

```diff
@@ -201,25 +202,51 @@ def rnnt_loss_simple(
         [0, 0, S, T]
         if boundary is not supplied.
         Most likely you will want begin_symbol and begin_frame to be zero.
+      reduction:
+        Specifies the reduction to apply to the output: `none`, `mean` or `sum`.
+        `none`: no reduction will be applied.
+        `mean`: apply `torch.mean` over the batches.
+        `sum`: the output will be summed.
+        Default: `mean`
       return_grad:
         Whether to return grads of px and py, this grad standing for the
         occupation probability is the output of the backward with a
         `fake gradient` input (all ones)  This is useful to implement the
         pruned version of rnnt loss.
     Returns:
-       If return_grad is False, returns a Tensor of shape (B,), containing the
-       NEGATED total RNN-T loss values for each element of the batch
-       (like log-probs of sequences).
+       If return_grad is False, returns a tensor of shape (B,), containing the
+       total RNN-T loss values for each element of the batch if reduction equals
+       to "none", otherwise a scalar with the reduction applied.
```

Comment:
Around line 214 we should clarify that the "fake gradient" is the same as the gradient you'd
get if you did `torch.grad( (-loss.sum()), [px, py])`.   
Now there is a - sign.
When you refer to the fake gradient below, you can add "(see above)" to clarify that it's explained above.